update_images

```diff
@@ -14,33 +14,33 @@ nprocs="${1:-1}"
 
 declare args
 
-while read -r line; do
-    IFS=',' read -r os release <<< "$line"
 
-    if [[ "${os}" = 'debian' ]] || [[ "${os}" = 'ubuntu' ]]; then
-        tag="${os}-${release}-all"
+IFS=',' read -r os release <<< "${TARGET_PLATFORM}"
```

Comment:
why do we no longer have a loop here? I do not like the fact that a script called `update_images` updates the images for a single platform.

It was easier to use this script when we could update all images that are listed in `os-list.csv`

Reply:
It is very error prone. I had very long build times when I first try to move this script. Then I realized that I didn't set  the parameters properly and built all the images for just one image type
Therefore I can update the script name into update_image if you want since this script will never be used for all the images in the pipeline